src/sampling/mindistance.jl

```diff
@@ -37,15 +37,15 @@ MinDistanceSampling(α, ρ, δ, metric) = MinDistanceSampling(addunit(α, u"m"),
 
 MinDistanceSampling(α::T; ρ=T(0.65), δ=100, metric=Euclidean()) where {T} = MinDistanceSampling(α, ρ, δ, metric)
 
-function sample(rng::AbstractRNG, d::Domain, method::MinDistanceSampling)
+function sample(rng::AbstractRNG, obj::GeometryOrDomain, method::MinDistanceSampling)
```

Comment:
```suggestion
function sample(rng::AbstractRNG, obj::Union{Domain,Ball}, method::MinDistanceSampling)
```
There is a [method ambiguity](https://github.com/JuliaGeometry/Meshes.jl/actions/runs/11523593652/job/32081938951?pr=1121#step:6:621). Is there a better choice than `Union{Domain,Ball}`?

Reply:
What's the advantage of your last [commit](https://github.com/JuliaGeometry/Meshes.jl/pull/1121/commits/7764ce9e6684c59669e5e820ec2fec08eed12cd3) compared to the `Union{Domain,Ball}` solution?